Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditquerylog/nhinc/proxy/AuditQueryLogProxyWebServiceUnsecuredImpl.java

```diff
@@ -0,0 +1,173 @@
+ */
+package gov.hhs.fha.nhinc.auditquerylog.nhinc.proxy;
+
+import gov.hhs.fha.nhinc.audit.retrieve.AuditRetrieve;
+import gov.hhs.fha.nhinc.auditquerylog.nhinc.proxy.service.QueryAuditEventsUnsecuredServicePortDescriptor;
+import gov.hhs.fha.nhinc.auditquerylog.nhinc.proxy.service.QueryByIdUnsecuredServicePortDescriptor;
+import gov.hhs.fha.nhinc.auditquerylog.nhinc.proxy.service.QueryByMessageIdAndRelatesToUnsecuredServicePortDescriptor;
+import gov.hhs.fha.nhinc.common.auditquerylog.QueryAuditEventsBlobRequest;
+import gov.hhs.fha.nhinc.common.auditquerylog.QueryAuditEventsBlobResponse;
+import gov.hhs.fha.nhinc.common.auditquerylog.QueryAuditEventsRequestByRequestMessageId;
+import gov.hhs.fha.nhinc.common.auditquerylog.QueryAuditEventsRequestType;
+import gov.hhs.fha.nhinc.common.auditquerylog.QueryAuditEventsResponseType;
+import gov.hhs.fha.nhinc.common.nhinccomponentauditquerylog.AuditQueryLogPortType;
+import gov.hhs.fha.nhinc.connectmgr.ConnectionManagerException;
+import gov.hhs.fha.nhinc.messaging.client.CONNECTCXFClientFactory;
+import gov.hhs.fha.nhinc.messaging.client.CONNECTClient;
+import gov.hhs.fha.nhinc.messaging.service.port.ServicePortDescriptor;
+import gov.hhs.fha.nhinc.nhinclib.NhincConstants;
+import gov.hhs.fha.nhinc.nhinclib.NullChecker;
+import gov.hhs.fha.nhinc.webserviceproxy.WebServiceProxyHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Webservices client implementation to search Audit events
+ *
+ * @author achidamb
+ */
+public class AuditQueryLogProxyWebServiceUnsecuredImpl implements AuditRetrieve {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AuditQueryLogProxyWebServiceUnsecuredImpl.class);
+    private final WebServiceProxyHelper proxyHelper = new WebServiceProxyHelper();
+    //method names to invoke
+    private final String QUERY_AUDIT_EVENTS = "queryAuditEvents";
+    private final String QUERY_AUDIT_EVENTS_BY_MSG_RELATESTO_ID = "queryAuditEventsByMessageID";
+    private final String QUERY_AUDIT_EVENTS_BY_ID = "queryAuditEventsBlob";
+
+    /**
+     *
+     * @param request - Request provides search params to retrieve Audit Events. If none of the elements are provided in
+     * request. If optional elements are not provided will return all audit events
+     * @return QueryAuditEventsResponseType
+     */
+    @Override
+    public QueryAuditEventsResponseType retrieveAudits(QueryAuditEventsRequestType request) {
+
+        QueryAuditEventsResponseType response = null;
+        try {
+
+            CONNECTClient<AuditQueryLogPortType> client = getCONNECTClient(getQueryAuditEventsPortDescriptor());
+            response = (QueryAuditEventsResponseType) client.invokePort(AuditQueryLogPortType.class,
+                QUERY_AUDIT_EVENTS, request);
+
+        } catch (Exception ex) {
+            LOG.error("Failed to call the web service {} {}", NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME
```

Comment:
You have only one parameter (NhincConstants.ADAPTER_AUDIT_QUERY_LOG_SERVICE_NAME + ex.getLocalizedMessage()) -- change the plus to a comma.